bitnami/postgresql/README.md

```diff
@@ -205,6 +205,7 @@ The following tables lists the configurable parameters of the PostgreSQL chart a
 | `metrics.image.pullPolicy`                    | PostgreSQL Exporter Image pull policy                                                                                                                                    | `IfNotPresent`                                                |
 | `metrics.image.pullSecrets`                   | Specify Image pull secrets                                                                                                                                                | `nil` (does not add image pull secrets to deployed pods)      |
 | `metrics.customMetrics`                       | Additional custom metrics                                                                                                                                                 | `nil`                                                         |
+| `metrics.extraEnvVars`                        | Extra environment variables to add to exporter  | `nil`    |
```

Comment:
We should mention that it's being evaluated as a template (you're using  "postgresql.tplValue" function):
```suggestion
| `metrics.extraEnvVars`                        | Extra environment variables to add to exporter  | `{}` (evaluated as a template)   |
```